packages/components/managed-data-table/src/index.js

```diff
@@ -0,0 +1,3 @@
+export { default } from './managed-data-table';
+export { useRowSelection, useSorting } from '@commercetools-uikit/hooks';
+export { default as tableSettingsActions } from './constants';
```

Comment:
What are these constants used for?

Reply:
This constants are used to identify the type of action that has been triggered, as we have a single function handling the change over the table settings. Parent component use this constants for example like this:

```
    const tableSettingsChangeHandler = {
      [tableSettingsActions.COLUMNS_UPDATE]: (visibleColumnKeys) =>
        setTableData({
          ...tableData,
          visibleColumnKeys,
        }),
      [tableSettingsActions.IS_TABLE_CONDENSED_UPDATE]: setIsCondensed,
      [tableSettingsActions.IS_TABLE_WRAPPING_TEXT_UPDATE]: setIsWrappingText,
    };
```

Reply:
Would it be helpful to have different callback handlers instead?

```js
onColumsUpdate
onTableCondenseUpdate
onTableWrappingTextUpdate
```

Then users can pick whatever handlers they need instead of having to map update actions.

Reply:
That was my initial approach but I was suggested to change it to only a reducer

Reply:
Hmm, I'm not familiar enough with the internals and the requirements so I can't say much about it. I just pointed this out because at a first glance it seemed that the user has to do some work to handle the callback. And also to avoid exporting (internal) constants.

Reply:
@tdeekens can you give me your thoughts on this? I'm ok both ways but as you [mentioned](https://github.com/commercetools/ui-kit/pull/1416#discussion_r452067115) having only one function would be a good option I would prefer to have an agreement with both of you

Reply:
Let's keep one for now. It might bring a bit of complexity here but eases the adoption and state management when consuming the table. As you don't have to wire things to multiple atoms or handlers.

Reply:
Should we have a more clear name maybe? Like `UPDATE_ACTIONS`.